Sources/Subs-Db-mysql.php

```diff
@@ -172,11 +172,9 @@ function smf_db_replacement__callback($matches)
 	if ($matches[1] === 'db_prefix')
 		return $db_prefix;
 
-	if ($matches[1] === 'query_see_board')
-		return $user_info['query_see_board'];
-
-	if ($matches[1] === 'query_wanna_see_board')
-		return $user_info['query_wanna_see_board'];
+	$user_queries = array_values(preg_grep('/^query_/', array_keys($user_info)));
```

Comment:
No need to use a regex here. It'd be much more efficient to use strpos() in this case.